src/sentry/api/serializers/models/organization_member/expand/projects.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any, Mapping, MutableMapping, Sequence, cast
```

Comment:
@mgaeta why put these serializers in another subdirectory? I think its ok to keep these files 1 level up since they're exported by `src/sentry/api/serializers/models/organization_member/__init__.py`